improve integration docs

hwchase17 opened the pull request.

hwchase17 committed:
improve integration docs

baskaryan commented on docs/docs/contributing/how_to/integrations/index.mdx:
```diff
@@ -60,18 +61,52 @@ new integrations in the docs of the following kinds:
-that will render on this site (https://python.langchain.com/).
+In order to contribute an integration, you should follow these steps:
+
+1. Confirm that your integration is in the [list of components](#components-to-integrate) we are currently encouraging.
```
encouraging -> accepting?

baskaryan commented on docs/docs/contributing/how_to/integrations/index.mdx:
```diff
@@ -60,18 +61,52 @@ new integrations in the docs of the following kinds:
+
+Each type of integration has its own set of standard tests. Please see the correct guide for setting up standard tests for your integration:
+
+- [Chat Models]
```
assume this is wip?

baskaryan commented on docs/docs/contributing/how_to/integrations/index.mdx:
```diff
@@ -60,18 +61,52 @@ new integrations in the docs of the following kinds:
-2. [Implement your package](./package.mdx) and publish it to a public github repository.
-3. [Implement the standard tests](./standard_tests) for your integration and successfully run them.
-4. [Publish your integration](./publish.mdx) by publishing the package to PyPi and add docs in the `docs/docs/integrations` directory of the LangChain monorepo.
+While traditionally called "co-marketing", we like to thing of this more as "co-education".
```
```suggestion
While traditionally called "co-marketing", we like to think of this more as "co-education".
```

ccurme committed:
docs[patch]: reorganize integration guides (#28457)

Proposal is:

- Each type of component (chat models, embeddings, etc) has a dedicated
guide.
- This guide contains detail on both implementation and testing via
langchain-tests.
- We delete the monolithic standard-tests guide.

ccurme committed:
Update docs/docs/contributing/how_to/integrations/index.mdx

Co-authored-by: Bagatur <22008038+baskaryan@users.noreply.github.com>

ccurme committed:
Merge branch 'master' into harrison/improve-integration-docs

ccurme committed:
docs[patch]: add to chat model contributing guide (#28490)

ccurme committed:
docs[patch]: add vector store contribution guide (#28518)

ccurme committed:
nit

ccurme committed:
Merge branch 'master' into harrison/improve-integration-docs

ccurme commented:
Copied new content to https://github.com/langchain-ai/langchain/pull/28547. Closing in favor of that and subsequent PRs.
